trento/adoc/trento-lifecycle.adoc

```diff
@@ -22,7 +22,7 @@ Delivery mechanisms::: A set of container images from the {suse} public registry
 {k8s} deployment::: The {trserver} runs on any current Cloud Native Computing Foundation (CNCF)-certified {k8s} distribution based on a x86_64 architecture. Depending on your scenario and needs, {suse} supports several usage scenarios:
 
 * If you already use a CNCF-certified {k8s} cluster, you can run the {trserver} in it.
-* If you don't have a {k8s} cluster, and need enterprise support, {suse} recommends {suse} {rke} (RKE) version 1 or 2.
+* If you don't have a {k8s} cluster, and need enterprise support, {suse} recommends {rke} version 1 or 2.
```

Comment:
We should remove any mentions of RKE1, see

```
Rancher Kubernetes Engine (RKE/RKE1) will reach end of life on July 31, 2025. Rancher 2.12.0 and later will no longer support provisioning or managing downstream RKE1 clusters
```` 

[Source](https://rke.docs.rancher.com/)